test/Microsoft.ML.CodeAnalyzer.Tests/TypeIsSchemaShapeTest.cs

```diff
@@ -78,15 +78,15 @@ public void ReturnTypeIsSchemaShapeClass()
             var diagPc = analyzer.SupportedDiagnostics[7];
 
             var expected = new DiagnosticResult[] {
-                diagF.CreateDiagnosticResult(32, 13, "Class4", "F1"),
-                diagC.CreateDiagnosticResult(33, 13, "Class5"),
-                diagC.CreateDiagnosticResult(34, 13, "Class6"),
-
-                diagPnas.CreateDiagnosticResult(35, 13, "Class7", "F1"),
-                diagPs.CreateDiagnosticResult(36, 13, "Class8", "F2"),
-                diagPg.CreateDiagnosticResult(37, 13, "Class9", "F2"),
-                diagPc.CreateDiagnosticResult(38, 13, "Class10"),
-                diagPc.CreateDiagnosticResult(39, 13, "Class11"),
+                diagF.CreateDiagnosticResult(33, 13, "Class4", "F1"),
+                diagC.CreateDiagnosticResult(34, 13, "Class5"),
+                diagC.CreateDiagnosticResult(35, 13, "Class6"),
+
+                diagPnas.CreateDiagnosticResult(36, 13, "Class7", "F1"),
+                diagPs.CreateDiagnosticResult(37, 13, "Class8", "F2"),
+                diagPg.CreateDiagnosticResult(38, 13, "Class9", "F2"),
+                diagPc.CreateDiagnosticResult(39, 13, "Class10"),
+                diagPc.CreateDiagnosticResult(40, 13, "Class11"),
```

Comment:
I was scrolling thru, getting carpal tunnel, about to conclude "oh look an easy PR approval". 

What are these changing in response to?

Reply:
This is our C# diagnostics analyzer test, which ensures our code analyzer gives the right warnings/errors on the correct code lines.

Because the code being analyzed used an ‘IDataView’ variable, I needed to insert a new ‘using’ statement at the top of the code. This moved all the code down one line, causing the tests to now be off by one line.